.github/workflows/docs.yml

```diff
@@ -80,24 +81,27 @@ jobs:
           yarn build
 
       # Start deploy --------------------------------------------------
+      - name: Copy asf file
+        run:  cp .asf.yaml ./website/build/.asf.yaml
+
       - name: Deploy to gh-pages
         uses: peaceiris/actions-gh-pages@v3.9.2
         if: github.event_name == 'push' && github.ref_name == 'main'
         with:
           github_token: ${{ secrets.GITHUB_TOKEN }}
-          cname: opendal.apache.org
           publish_dir: website/build
           publish_branch: gh-pages
 
-      - name: Deploy to vercel (preview)
-        if: github.repository_owner == 'datafuselabs' && github.ref_name != 'main'
-        uses: amondnet/vercel-action@v25.1.1
-        with:
-          vercel-token: ${{ secrets.VERCEL_TOKEN }}
-          github-token: ${{ secrets.GITHUB_TOKEN }}
-          vercel-org-id: ${{ secrets.VERCEL_ORG_ID }}
-          vercel-project-id: ${{ secrets.VERCEL_PROJECT_ID }}
-          alias-domains: |
-            opendal-git-{{BRANCH}}.vercel.app
-          scope: databend
-          working-directory: ./website/build
+      # Disable until we are allowed
```

Comment:
I don't think we will use Vercel for preview. I don't find any ASF projects setting up this before. Could you file an issue on INFRA and associate it here?